src/networkWrapper/wrappers/websocket.js

```diff
@@ -84,113 +75,34 @@ function WSNode(host, port, ssl) {
    * @param {function} callback
    */
   this.onConnect = function (callback) {
-    this.listeners.connect.push({
-      fn: callback,
-      keep: true
-    });
+    this.addListener('connect', callback);
   };
 
   /**
    * Fires the provided callback whenever a connection error is received
    * @param {function} callback
    */
   this.onConnectError = function (callback) {
-    this.listeners.error.push({
-      fn: callback,
-      keep: true
-    });
+    this.addListener('wserror', callback);
```

Comment:
`error` events have a special behaviour in Node, and throw an error if it is triggered with no listener (see https://nodejs.org/api/events.html#events_error_events )
So we need another event name for Kuzzle errors to avoid bad behaviour.

Looking for a better name, any suggestion is welcome ;)